charts/rancher-monitoring/101.1.0+up40.1.2/charts/prometheus-node-exporter/templates/servicemonitor.yaml

```diff
@@ -47,5 +47,17 @@ spec:
     {{- with .Values.prometheus.monitor.metricRelabelings }}
       metricRelabelings:
         {{- toYaml . | nindent 8 }}
+        {{ if .Values.global.cattle.clusterId }}
```

Comment:
same comment as above regarding moving `metrics-relabeling` outside the `if` statement.